configs/vision/pathology/online/segmentation/consep.yaml

```diff
@@ -79,11 +79,17 @@ model:
         - class_path: torchmetrics.ClasswiseWrapper
           init_args:
             metric:
-              class_path: eva.vision.metrics.GeneralizedDiceScore
+              class_path: eva.vision.metrics.MonaiDiceScore
               init_args:
+                include_background: true
                 num_classes: *NUM_CLASSES
-                weight_type: linear
-                per_class: true
+                reduction: none
+            labels:
+              - background
+              - other
+              - inflammatory
+              - epithelial
+              - spindle-shaped
```

Comment:
are these all labels for consep? in the doc we mention 7 nuclei types: https://kaiko-ai.github.io/eva/main/datasets/consep/

Reply:
good catch, updated the docs now